src/combinatorics/configuration.lean

```diff
@@ -413,11 +413,150 @@ variables {P} (L)
 lemma two_lt_line_count [projective_plane P L] (p : P) : 2 < line_count L p :=
 by simpa only [line_count_eq L p, nat.succ_lt_succ_iff] using one_lt_order P L
 
+-- better name?
+lemma nontrivial_lines_through_point [projective_plane P L] (p : P) :
+  nontrivial {l : L // p ∈ l} :=
```

Comment:
Maybe define `pencil p := {l : L // p ∈ l}`? And then this is called `nontrivial_pencil`. And I guess `line l := {p : P // p ∈ l}`?

Reply:
Seems reasonable, but if I can get the shorter proof to work, then we might not need this.

Reply:
Ok, sounds good! Nevertheless,I suggest introducing it unrelated to this PR. And then `line_count` and `point_count` could be renamed to `card_pencil` and `card_line`.